webview/src/experiments/components/table/content/Header.tsx

```diff
@@ -9,7 +9,7 @@ interface HeaderProps {
 export const Header: React.FC<HeaderProps> = ({ name }) => {
   return (
     <OverflowHoverTooltip content={name}>
-      <div className={styles.headerCellWrapper}>
+      <div className={styles.headerCellText}>
```

Comment:
Maybe it's just me, but `headerCellWrapper` gives me the impression that the class wraps the cell when in reality, it's inside the cell, wrapping the text. Renamed it to `headerCellText`.